Tests/ML/test_metrics.py

```diff
@@ -106,12 +116,16 @@ def generate_random_prediction() -> np.ndarray:
 
     random_prediction = generate_random_prediction()
     matching_gt = np.stack([1 - random_prediction, random_prediction])
-    assert_metrics(random_prediction, matching_gt, 0.0)
+    # Since there is only label, 'missing_labels' list to have only element
+    missing_labels_list = [False]
+    assert_metrics(random_prediction, matching_gt, missing_labels_list, 0.0)
     # check voxel spacing is being used as expected
     random_prediction2 = generate_random_prediction()
     non_matching_gt = np.stack([1 - random_prediction2, random_prediction2])
-    without_spacing = assert_metrics(random_prediction, non_matching_gt, voxel_spacing=(1, 1, 1), expected=None)
-    with_spacing = assert_metrics(random_prediction, non_matching_gt, voxel_spacing=(2.0, 2.0, 2.0), expected=None)
+    without_spacing = assert_metrics(random_prediction, non_matching_gt, missing_labels_list,
+                                     voxel_spacing=(1, 1, 1), expected=None)
+    with_spacing = assert_metrics(random_prediction, non_matching_gt, missing_labels_list,
+                                  voxel_spacing=(2.0, 2.0, 2.0), expected=None)
```

Comment:
Can you get your line length settings in line with the rest of the team? Such spurios changes bloat PR.